### PR TITLE
A few working functions and unit tests

### PR DESCRIPTION
**Pull request by cartemic**
5 functions: 3 in tools, and 2 in accessories. Tests are split among appropriate files in test directory. 100% pass on linux and win7, and pylint is currently happy. Each directory has a separate .pylintrc file, because some of the tests needed relaxed linting standards due to function mocking.

### CODE REVIEW

File: TubeDesignTools/tools.py [hunk 1]
```diff
@@ -353,4 +261,4 @@ def get_spiral_diameter(pipe_id, blockage_ratio):
 
 
 if __name__ == '__main__':
```
Comment (derekb63) [inline TubeDesignTools/tools.py]: I am not sure if you want to include the if name main block in the actual files. I haven't seen those when looking at actual package files

File: Tests/test_tools.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    """
+    # ----------------------------INPUT TESTING----------------------------
+    # check for error handling with non-string material
+    # check for error handling with bad material
+
+    # incorporate units with pint
+    ureg = pint.UnitRegistry()
+    quant = ureg.Quantity
+
+    # set temperatures
```
Comment (derekb63) [inline Tests/test_tools.py]: Maybe add some comments about why you chose the limits for the temperatures and pressures

File: TubeDesignTools/tools.py (hunk 1 above)
Comment (cartemic) [reply TubeDesignTools/tools.py]: Good call, this has been removed.

File: Tests/test_tools.py (hunk 2 above)
Comment (cartemic) [reply Tests/test_tools.py]: Done.

File: beaverdet/tests/test_accessories.py [hunk 3]
```diff
@@ -0,0 +1,779 @@
...
+                'asdfflangegroup1asd',
+                'asdfstressweldedasdg']
+
+    # run test suite
+    with patch('builtins.open', new=FakeOpen):
+        print()
+        print()
+        print(accessories.__file__)
+        print()
+        print()
```
Comment (derekb63) [inline beaverdet/tests/test_accessories.py:98]: Remove these print statements

**Review by derekb63 — APPROVED** [review]

**Pull request merged**